Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/callback/openSAML/SVSAMLAssertionBuilder.java

```diff
@@ -44,7 +44,7 @@
      * @see gov.hhs.fha.nhinc.callback.openSAML.SAMLAssertionBuilder#build()
      */
     @Override
-    public Element build(CallbackProperties properties) throws Exception {
+    public Element build(CallbackProperties properties) throws SAMLAssertionBuilderException {
```

Comment:
Functions that behave differently than their signature or JavaDoc would suggest should have comments explaining why (for example, SAMLAssertionBuilderException is not thrown, and nothing is actually built).